React/tic-tac/src/app/screens/Login/style.scss

```diff
@@ -0,0 +1,106 @@
+.login {
+  background-color: #F8F8F8;
+  border-radius: 5px;
+  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.1);
+  left: -15px;
+  margin: 0 auto;
+  padding: 4em;
+  position: fixed;
+  right: -15px;
+  top: 200px;
+  width: 30%;
+
+  .space {
```

Comment:
If this `.space` class is of the form styles, move it to the `styles.scss` file of the form component.